nuxeo-web-ui-ftest/test/features/step_definitions/support/fixtures/documents.js

```diff
@@ -107,7 +106,8 @@ module.exports = function () {
   this.Before(() => nuxeo.repository().fetch('/default-domain').then((doc) => { this.doc = doc; }));
 
   this.After(() => Promise.all(liveDocuments
-      .filter((doc) => path.dirname(doc) === '/default-domain')
-      .map((doc) => nuxeo.repository().delete(doc)))
-      .then(() => { liveDocuments = []; }));
+      .map((docUid) => nuxeo.repository().delete(docUid)))
```

Comment:
We should do the catch on the delete instead, otherwise the exception will interrupt the iteration and some documents may not be deleted.
```
.map((docUid) => nuxeo.repository().delete(docUid).catch(() => {}))) // eslint-disable-line arrow-body-style
```